src/xclim/indicators/atmos/_precip.py

```diff
@@ -815,3 +816,15 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.water_cycle_intensity,
 )
+
+antecedent_precipitation_index = Indicator(
+    title="Antecedent Precipitation Index",
+    identifier="antecedent_precipitation_index",
```

Comment:
```suggestion
    identifier="api",
```
Your call, but seeing that the indicator seems well known, we can use it's common identifier in xclim as well.